packages/core/src/cli.ts

```diff
@@ -163,7 +180,21 @@ prog
     }
 
     if (opts.client) {
-      await boostrapCodegen(opts.schema, true)
+      if (!(await isUsingGraphQLTada(baseDirectory))) {
+        await boostrapCodegen(opts.schema, true)
+      } else {
+        const hasSrcDir = existsSync(path.resolve(baseDirectory, 'src'))
+        const base = hasSrcDir
+          ? path.resolve(baseDirectory, 'src')
+          : baseDirectory
+        await Promise.allSettled([
+          fs.writeFile(
+            path.resolve(base, 'fuse/index.ts'),
+            `// This is a generated file!\n\nexport * from './tada';\nexport * from 'fuse/client';\n`,
+          ),
+          fs.writeFile(path.resolve(base, 'fuse/tada.ts'), tadaGqlContents),
```

Comment:
In the future when folks use custom scalars, ... we might not want to override this